scripts/install/run.sh

```diff
@@ -0,0 +1,337 @@
+#!/usr/bin/env bash
+set -e
+
+# Prints header
+function printh {
+    local input="$1"
+    local len=$((${#1}+2))
+    printf "\033[32m" # start green color
+    printf "\n+"
+    printf -- "-%.0s" $(seq 1 $len)
+    printf "+\n| $input |\n+"
+    printf -- "-%.0s" $(seq 1 $len)
+    printf "+\n\n"
+    printf "\033[0m" # end green color
+}
+
+function integrate_with_smartedit {
+  cmd_install "$@"
+
+  cp ${CLONE_DIR}/projects/storefrontapp/src/webApplicationInjector.js ${APP_DIR}/src
+
+  jq ".projects.${APP_NAME}.architect.build.options.assets += [\"src/webApplicationInjector.js\"]" ${APP_DIR}/angular.json > ${APP_DIR}/temp.json
+
+  mv ${APP_DIR}/temp.json ${APP_DIR}/angular.json
+}
+
+function delete_dir {
+    if [ -d "${1}" ]; then
+        echo "deleting ./${1}"
+        rm -rf ${1}
+    fi
+}
+
+function cmd_clean {
+    printh "Cleaning Spartacus installation workspace"
+
+    delete_dir $CLONE_DIR
+    delete_dir $INSTALLATION_DIR
+    delete_dir storage
+}
+
+function pre_install {
+    if [ -d $BASE_DIR ]; then
+        echo "Directory ${BASE_DIR} already exists, please remove it in order to start fresh installation."
+        exit 1
+    fi
+    VERDACCIO_PID=`lsof -nP -i4TCP:4873 | grep LISTEN | tr -s ' ' | cut -d ' ' -f 2`
+    if [[ -n ${VERDACCIO_PID} ]]; then
+        echo "It seems verdaccio is already running on PID: ${VERDACCIO_PID}. Please kill it first to continue."
```

Comment:
Is there a reason why this can happen? Is it common?

Reply:
If something goes wrong between starting verdaccio and killing it, next run of the script used to have problems with already running verdaccio (with some of libs already published). What do you think of automatically killing it instead of exit?

Reply:
It happens in dev-servers sometimes.

Reply:
I would kill it